app/models/billing-detail.js

```diff
@@ -1,10 +1,19 @@
 import DS from 'ember-data';
+import Ember from 'ember';
 
 export default DS.Model.extend({
-  plan: DS.attr(),
-  paymentMethodName: DS.attr(),
-  paymentMethodDisplay: DS.attr(),
+  plan: DS.attr('string'),
+  paymentMethodName: DS.attr('string'),
+  paymentMethodDisplay: DS.attr('string'),
   nextInvoiceDate: DS.attr('iso-8601-timestamp'),
-  paymentExpMonth: DS.attr('number'),
-  paymentExpYear: DS.attr('number')
+  //paymentExpMonth: DS.attr('number'),
+  //paymentExpYear: DS.attr('number'),
+  allowPHI: Ember.computed.match('plan', /production|platform/),
+  stripeSubscriptionId: DS.attr('string'),
+  stripeCustomerId: DS.attr('string'),
+  hasStripeSubscription: Ember.computed.bool('stripeSubscriptionId'),
+  hasStripeCustomer: Ember.computed.bool('stripeCustomerId'),
+  hasStripe: Ember.computed.and('hasStripeCustomer', 'hasStripeSubscription'),
+  organization: DS.belongsTo('organization', {async:true})
+  //stripeSubscriptionStatus: DS.attr()
```

Comment:
Also, we usually group model attributes and relationships before normal props

```js
export default DS.Model.extend({
  plan: DS.attr('string'),
  foo: DS.attr('number'),
  organizations: DS.hasMany('organization', {async: true}),

  hasSomething: Ember.computed.bool('someThing')
});
```

It must make it slightly easier to reason about and understand the schema of the model.